bin/pre-push-hook.js

```diff
@@ -11,8 +11,8 @@ console.log(
 
 const currentBranch = execSync( 'git rev-parse --abbrev-ref HEAD' ).toString().trim();
 
-if ( 'master' === currentBranch ) {
-	if ( ! readline.keyInYN( "You're about to push !!![ master ]!!!, is that what you intended?" ) ) {
+if ( 'trunk' === currentBranch ) {
```

Comment:
Do you think we need to support `trunk` and `master` for a period of time while we wait for everybody to rename their local branches?

Reply:
Ah, that's a good idea. That should also be helped by the `calypso-doctor` check I'm going to add as well.